stdlib/strings/strings_test.flux

```diff
@@ -1,198 +1,302 @@
 package strings_test
 
-import "testing"
+import "array"
 import "strings"
+import "testing"
 
 option now = () => (2030-01-01T00:00:00Z)
 
-inData = "
-#datatype,string,long,dateTime:RFC3339,string,string,string,string,string,string,string
-#group,false,false,false,false,true,true,true,true,true,true
-#default,_result,,,,,,,,,
-,result,table,_time,_value,_field,_measurement,device,fstype,host,path
-,,0,2018-05-22T19:53:26Z,字,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:36Z,k9n  gm,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:46Z,b  ,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:56Z,2COTDe,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:54:06Z,cLnSkNMI  ,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:54:16Z,13F2  ,used_percent,disk,disk1,apfs,host.local,/
-"
-
+// leading and trailing whitespace is removed from a string
 testcase string_trim {
-    outData = "
-#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,string,string,string,string
-#group,false,false,true,true,false,false,true,true,true,true,true,true
-#default,_result,,,,,,,,,,,
-,result,table,_start,_stop,_time,_value,_field,_measurement,device,fstype,host,path
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,字,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:36Z,k9n  gm,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,b,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:56Z,2COTDe,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:06Z,cLnSkNMI,used_percent,disk,disk1,apfs,host.local,/
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:16Z,13F2,used_percent,disk,disk1,apfs,host.local,/
-"
-    result = testing.loadStorage(csv: inData)
-		|> range(start: 2018-05-22T19:53:26Z)
-		|> map(fn: (r) =>
-        			({r with _value: strings.trimSpace(v: r._value)}))
-    testing.diff(got: result, want: testing.loadMem(csv: outData))
+    input = array.from(rows: [{
+        _time: 2018-05-22T19:53:26Z,
+        _value: "trailing  ",
+        _field: "_field",
+        _measurement: "_measurement",
+    }, {
+        _time: 2018-05-22T19:53:36Z,
+        _value: "   leading",
+        _field: "_field",
+        _measurement: "_measurement",
+    }])
+    // XXX: rockstar (31 Mar 2021) - Adding `map` here, otherwise `testing.diff`
+    // gets a type error because it thinks it's missing a _value label.
```

Comment:
Why do so many test cases think they're missing a `_value` label? Is this something we need to fix?

Reply:
I was just on the phone with @nathanielc, and it turns out this is a actual problem we have, with a Big Kid Issue™ and everything. https://github.com/influxdata/flux/issues/3443

I'm going to add that reference to the comments, so there is context there.